docs/developer/modules/03-merger.md

```
@@ -0,0 +1,106 @@
+
+What this module is responsible for:
+
+- Execute processors in parallel via HTTP
+- Merge processor outputs into single directory
+- Execute plugins sequentially on merged output
+- Resolve processor and plugin versions from registry
+- Handle errors and collect results
+
+## Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)